src/server/hook_func.c

```diff
@@ -3564,6 +3564,10 @@ do_runjob_reject_actions(job *pjob, char *hook_name)
 		}
 	}
 
+	/* update the eligible time to JOB_INITIAL */
+	if (server.sv_attr[(int)SRV_ATR_EligibleTimeEnable].at_val.at_long == 1)
```

Comment:
Check if ATR_VFLAG_SET is set as well

Reply:
There were existing instances of this which don't check for ATR_VFLAG_SET, do you think we should always do that check? As such, this value should be set to 0 when we unset it, or if it's not set right?